.changelog/unreleased/features/ibc-telemetry/2381-configure-wallet-balance-divider.md

```diff
@@ -0,0 +1,3 @@
+- Added a new configuration, `wallet_metric_scale`, to `config.toml` to specify the
+  value used to divide the `wallet_balance` displayed in the telemetry metrics.
```

Comment:
```suggestion
  value used to scale down the `wallet_balance` displayed in the telemetry metrics
  in case the raw value does not fit into a unsigned 64-bits integer,
  eg. because the relayer wallet contains a high amount of coins.
```